vdev/src/testing/integration.rs

```diff
@@ -32,6 +33,9 @@ impl IntegrationTest {
         build_all: bool,
         retries: u8,
     ) -> Result<Self> {
+        // some tests require the rust version env var to be set
+        export_rust_version()?;
```

Comment:
This seems like a poor place to put this. It could be added to the config's environment variables, perhaps, or maybe set up in the runner somewhere.

Reply:
Good call. I put it down where the service environment commands are run, since it needs to be in the environment for the services.